src/clients/extension-manager.js

```diff
@@ -65,6 +70,33 @@ export async function getPlatforms() {
   return await jsonApi.get(url);
 }
 
+export async function uploadPlatform(tgzStream, progressHandler, size) {
+  // a temporary workaround, forces access token to refresh
+  await getDeveloper();
```

Comment:
@aesqe Can we use fetch token intercept here instead of these superfluous calls?